examples/express/connection-sharing/app.js

```diff
@@ -1,18 +1,15 @@
+"use strict";
```

Comment:
Per the package.json file, it looks like the coding standards for this repo indicate that single quotes should be used instead of double quotes?